src/pages/download.md

```diff
@@ -21,6 +21,19 @@ title: 软件下载
 <b>普通用户建议优先使用正式版。</b>专业用户以及愿意协助测试的用户可以尝试使用测试版。
 :::
 
+:::tip
+**测试版通过 GitHub Actions 进行分发，需要登录 GitHub 账号才能下载。**
+
+如果你没有或不方便注册/登录 GitHub，也可以通过 nightly.link 下载：
+
+- 桌面测试版：[https://nightly.link/lyswhut/lx-music-desktop/workflows/beta-pack/beta](https://nightly.link/lyswhut/lx-music-desktop/workflows/beta-pack/beta)
+- 移动测试版：[https://nightly.link/lyswhut/lx-music-mobile/workflows/beta-pack/beta](https://nightly.link/lyswhut/lx-music-mobile/workflows/beta-pack/beta)
+
+**nightly.link 仅指向最新的测试版本。**
+
+如果测试版版本号比正式版低，**请优先使用版本号更高的版本**。
```

Comment:
The version of the **universal** package is always lower than the version of the single-architecture package. When installing a new version, if you encounter a prompt that the version number is too low, try using the single-architecture installation package.